aggregate/quality_of_life/heat_vulnerability.py

```diff
@@ -1,28 +1,36 @@
 import pandas as pd
 from ingest.ingestion_helpers import add_leading_zero_PUMA
 from internal_review.set_internal_review_file import set_internal_review_files
+from utils.PUMA_helpers import get_all_boroughs
 
 
 def load_clean_heat_vulnerability(geography):
-    assert geography == "puma", "This indicator only calculated at PUMA level"
-    df = pd.read_excel(
-        "./resources/quality_of_life/HVI_PUMA_Subboro_forSharing.xlsx",
-        usecols=["PUMACE10", "HVI"],
-        dtype={"PUMACE10": str, "HVI": int},
-    )
-
-    df.rename(
-        columns={"PUMACE10": "puma", "HVI": "health_heat"},
-        inplace=True,
-    )
-
-    print(f"Successfully loaded heat vulnerability xlsx")
-
-    df = add_leading_zero_PUMA(df)
-
-    print(f"Add leading zero to puma column")
-
-    return df
+    assert geography in ["puma", "borough", "citywide"]
+    if geography == "puma":
+        df = pd.read_excel(
+            "./resources/quality_of_life/HVI_PUMA_Subboro_forSharing.xlsx",
+            usecols=["PUMACE10", "HVI"],
+            dtype={"PUMACE10": str, "HVI": int},
+        )
+
+        df.rename(
+            columns={"PUMACE10": "puma", "HVI": "health_heat"},
+            inplace=True,
+        )
+
+        print(f"Successfully loaded heat vulnerability xlsx")
+
+        df = add_leading_zero_PUMA(df)
+        df = df.set_index("puma")
+        print(f"Add leading zero to puma column")
+
+        return df
+    elif geography == "borough":
+        return pd.DataFrame(
+            index=get_all_boroughs(), columns=["health_heat"], data=None
+        )
+    else:
+        return pd.DataFrame(index=["citywide"], columns=["health_heat"], data=None)
 
 
 def set_results_for_internal_review(df, geography):
```

Comment:
refactor to work with the collate function- thanks @SashaWeinstein

Reply:
yea felt good ab this change, didn't think it at all until I tried to iterate through each indicator/geography combination